### DISCUSSION TITLE
Invite billing user during setup of new Pro team

### DISCUSSION THREAD

**Discussion by simonkuran** [vercel/community#7318] [Help] [answered]
### Summary

Hi,

I'd like to create a new Pro team for a new project and invite our client as a billing user. Unfortunately, it looks like my account has used up its free trials. Is it possible for me to create a new team without charging my company's credit card before the client takes over the billing? This would be needlessly messy for us to invoice.

It feels like I should be able to create the team and invite a billing user. Once the billing user has added a credit card the team could become functional. The UX for the account setup when you can start a trial and have a client setup the billing in the next couple days is so much better than this. Is there a way to achieve this?

Thanks,
Simon

### Steps to Reproduce

1. Have 2+ Pro teams for other projects (so free your trials are used up).
2. Try to create a new Pro team.

**Comment by pawlean** [comment, marked as answer]
Hi @simonkuran!

Could you raise a support case [here](https://vercel.com/help)? If you scroll down to "Vercel Customer Support" then "Get Help" and follow the instructions. 

Thank you!

[a comment by an automated account is not shown]
